build/cli.js

```diff
@@ -243,18 +247,28 @@ async function buildOtherSPAs() {
   })();
 
   (() => {
-    const url = "/en-US/search";
-    const html = renderHTML(url);
-    // XXX Loop over all the locales distinct that are going to be built.
-    // XXX so it's not just en-US.
-    const outPath = path.join(BUILD_OUT_ROOT, "en-us", "search");
-    fs.mkdirSync(outPath, { recursive: true });
-    const filePath = path.join(outPath, "index.html");
-    fs.writeFileSync(filePath, html);
-    console.log("Wrote", filePath);
+    // Basically, this builds on `search/index.html` for every locale we intend
+    // to build.
+    for (const root of [CONTENT_ROOT, CONTENT_TRANSLATED_ROOT]) {
+      if (!root) {
+        continue;
+      }
+      for (const locale of fs.readdirSync(root)) {
+        if (!fs.statSync(path.join(root, locale)).isDirectory()) {
```

Comment:
Is there a reason why it has to be sync?
I'm curious. Perhaps worth to comment the background.
(Unrelated to this PR).

Reply:
Because it's faster.